packages/ember/tests/routing/decoupled_basic_test.js

```diff
@@ -79,14 +79,6 @@ moduleFor(
       });
     }
 
-    get currentPath() {
-      let currentPath;
-      expectDeprecation(() => {
-        currentPath = this.applicationInstance.lookup('controller:application').get('currentPath');
```

Comment:
...and again here, use the router service.

(I'm surprised these were not all refactored when the deprecation was added.)